lnd_test.go

```diff
@@ -10907,6 +10908,34 @@ func testSendUpdateDisableChannel(net *lntest.NetworkHarness, t *harnessTest) {
 		ctxt, t, net, net.Alice, carol, chanAmt, 0, false,
 	)
 
+	// We create a new node Eve that has an inactive channel timeout of
+	// just 5 secnods (down from the default 20m). it will be used to test
+	// channel updates for channels going inactive.
+	eve, err := net.NewNode("Eve", []string{"--inactivechantimeout=5s"})
```

Comment:
Why not a lower timeout? Seems possible that on a fast machine, the assertion below will fail as it runs through the logic below fast enough.

Reply:
It will wait up to 20 seconds for the update, so should be plenty of time. Did lower to `2s`.